vir/simd_execution.h

```diff
@@ -0,0 +1,256 @@
+/* SPDX-License-Identifier: LGPL-3.0-or-later */
+/* Copyright © 2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH
+ *                  Matthias Kretz <m.kretz@gsi.de>
+ */
```

Comment:
My consistency goal is to not add copyright and license to (new) *tests*. Because that's how it's done in libstdc++. If some tests already have a header that's because I did it "wrong" - and they're copies of files in libstdc++.

Reply:
Okay, this is somewhat confusing to me. I learned "no license/copyright" means that a user does not have *any* permissions.

On the other hand, you're project usually has a "project wide license" and that one can be assumed as a fallback license.

Anyhow, if that's the idea, understood.